### DISCUSSION TITLE
Export data to a different pimcore instance

### DISCUSSION THREAD

**Discussion by LouKordos** [pimcore/pimcore#17282] [Q&A / Support]
Hi, I am in the process of upgrading from pimcore 6.8 to Pimcore 11 and after that, I want to try moving my install to docker. Since I'm not familiar with pimcore and only managing the server that it's running on, I wanted to ask what I need to export and then import to get the same instance running in docker. Just copy over the whole db in mysql? or is there anything I need to look out for?

**Comment by benyaminl** [comment]
Read the docs 
https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/Updating_Pimcore/V6_to_V10/#ensure-your-database-configuration-includes-the-following-definitions 

After go to 10 then to 11, follow the migration path

**Comment by LouKordos** [comment]
I read those docs, I was talking about what I need to move between Servers
if I have two pimcore 11 installs but want to keep my data. Is a mysqldump
sufficient? Say I want a clone instance for staging.

**Reply by natted** [reply]
You can use mysqldump but I found you need to use the --routines flag
